.stylelintignore

```diff
@@ -0,0 +1,3 @@
+client/src/styles/legacy/*
```

Comment:
Why are we ignoring these?

Reply:
I took this list from the old config file, [see here](https://github.com/silverstripe/silverstripe-elemental-bannerblock/pull/141/files#diff-ef21bbf44f28384ec21752295db99461de5bf58b4bdca9375e3d709d32967c54L4-L7)

Reply:
I think since these files were ignored before, most likely they will not pass the `stylelint` test. Most likely, this point should be added to the next task.